Добавлено: Подключение аудиоинтерфейса и подключение нативных vst пла… (#655)
OlegShchavelev/ALTGnomeWiki: docs/apps/reaper.md

![Reaper](/reaper/reaper-1.png)

## Подключение аудиоинтерфейса

> В меню Options/Preferences необходимо перейти на вкладку Audio/Device и выбрать аудиосистему, нужный аудио интерфейс и расставить каналы. Для использования JACK необходимо дополнительно установить пакет pipewire-jack и qjackctl.

Reviewer: ```suggestion
В меню Options/Preferences необходимо перейти на вкладку Audio/Device и выбрать аудиосистему, нужный аудиоинтерфейс и расставить каналы. Для использования JACK необходимо дополнительно установить пакет `pipewire-jack` и `qjackctl`.
```

> Проверить работу интерфейса можно добавив новую дорожку Ctrl+T, выбрать нужный канал и нажать кнопку rec (Ctrl+R)

Reviewer: ```suggestion
Проверить работу интерфейса можно следующим образом: добавить новую дорожку [[Ctrl + T]], выбрать нужный канал и нажать кнопку «Rec» ([[Ctrl + R]])
```

> ![Reaper](/reaper/reaper-audio_setting.png)
> 
> 
> ## Подключение нативных vst плагинов

Reviewer: ```suggestion
![Reaper](/reaper/reaper-audio_setting.png)

## Подключение нативных VST-плагинов
```

> Для удобства советуется скачанные/приобретенные плагины положить в один каталог в распакованном виде. Далее в меню Options/Preferences необходимо перейти на вкладку Plug-ins/vst, нажать на кнопку "Edit path list..." и выбрать пункт "add patch" и указать путь к каталогу с плагинами. После того как новый путь был выбран необходимо нажать на кнопку "Re-scan..." и выбрать пункт "Clear cache" and re-scan VST paths for all plug-ins или перезагрузить reaper.

Reviewer: ```suggestion
Для удобства рекомендуется положить скачанные/приобретённые плагины в один каталог в распакованном виде. Далее в меню Options/Preferences необходимо перейти на вкладку Plug-ins/VST, нажать на кнопку «Edit path list...» и выбрать пункт «Add patch...» и указать путь к каталогу с плагинами. После того, как новый путь был выбран, необходимо нажать на кнопку «Re-scan...» и выбрать пункт «Clear cache and re-scan VST paths for all plug-ins» или перезагрузить Reaper.
```

> Важно! Нажав в меню "Edit path list..." на уже существующий путь, он удалится! 

Reviewer: ```suggestion
::: warning Важно!
Нажав в меню «Edit path list...» на уже существующий путь, он удалится!
:::
```

> Для плагинов в формате LV2 в меню Options/Preferences необходимо перейти на вкладку Plug-ins/LV2, нажать на кнопку "Edit path list..." и выбрать пункт "add patch" и указать путь к каталогу с плагинами. После того как новый путь был выбран необходимо нажать на кнопку "Re-scan...".

Reviewer: ```suggestion
Для плагинов в формате LV2 в меню Options/Preferences необходимо перейти на вкладку Plug-ins/LV2, нажать на кнопку «Edit path list...» и выбрать пункт «Add patch» и указать путь к каталогу с плагинами. После того, как новый путь был выбран необходимо нажать на кнопку «Re-scan...».
```

> ![Reaper](/reaper/reaper-plug-ins-1.png)
> ![Reaper](/reaper/reaper-plug-ins-2.png)

Reviewer: ```suggestion
![Reaper](/reaper/reaper-plug-ins-1.png)

![Reaper](/reaper/reaper-plug-ins-2.png)
```
